pkg/proto/CommonDTO.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.30.0
-// 	protoc        v3.21.12
+// 	protoc-gen-go v1.28.1
```

Comment:
@irfanurrehman Should I upgraded it?

Reply:
@andreiastra  you should use the same version of protoc-gen that you see in code to ensure we do not change the autogenerated code by going down the versions.

Reply:
Upgraded `protoc-gen-go`. The versions look good now.